lib/servers/Multiplexer/Multiplexerserver.py

```diff
@@ -505,6 +526,21 @@ def get_pid_polarity(self, c, dacPort):
                                        ctypes.pointer(polarity), dummyarg)
 
         returnValue(polarity.value)
+        
+    @setting(37, "get_wavemeter_pattern", chan='i')
+    def get_wavemeter_pattern(self, c, chan):
+        """
```

Comment:
Should this function return the wavemeter pattern too?

Reply:
I'm not sure it will ever be used but we have changed the function to return both interferometer patterns.